src/index.html

```diff
@@ -4,24 +4,454 @@
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+              </div>
+
+              <div>
+                <span class="card__review">Reviews: 5</span>
+              </div>
+            </div>
+
+            <div class="card__prices">
+              <span class="card__price text">Price:</span>
+              <span class="card__price value">$2,199</span>
+            </div>
+
+            <button class="button card__button">Buy</button>
+          </div>
+        </article>
+
+        <article class="card">
+          <div class="card__image">
+            <img
+              src="./images/imac.jpeg"
+              alt="imac"
+            />
+          </div>
+
+          <div class="card__info">
+            <h2 class="card__name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+            <h3 class="card__code">Product code: 195434</h3>
+
+            <div class="card__reviews">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+              </div>
+
+              <div>
+                <span class="card__review">Reviews: 5</span>
+              </div>
+            </div>
+
+            <div class="card__prices">
+              <span class="card__price text">Price:</span>
+              <span class="card__price value">$2,199</span>
+            </div>
+
+            <button class="button card__button">Buy</button>
+          </div>
+        </article>
+
+        <article class="card">
+          <div class="card__image">
+            <img
+              src="./images/imac.jpeg"
+              alt="imac"
+            />
+          </div>
+
+          <div class="card__info">
+            <h2 class="card__name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+            <h3 class="card__code">Product code: 195434</h3>
+
+            <div class="card__reviews">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+
+                <div class="stars__star"></div>
+              </div>
+
+              <div>
+                <span class="card__review">Reviews: 5</span>
+              </div>
+            </div>
+
+            <div class="card__prices">
+              <span class="card__price text">Price:</span>
+              <span class="card__price value">$2,199</span>
+            </div>
+
+            <button class="button card__button">Buy</button>
+          </div>
+        </article>
+      </section>
+    </main>
```

Comment:
I noticed that you have repeated the same block of code several times. This is not a good practice as it can make your code harder to maintain and read. Consider using a loop to generate these elements dynamically. If you are not using a JavaScript framework that supports this feature, you can use a server-side language or a static site generator.